src/backend/utils.ts

```diff
@@ -917,10 +917,10 @@ export async function downloadDefaultWine() {
   await updateWineVersionInfos(true)
   // get list of wines on wineDownloaderInfoStore
   const availableWine = wineDownloaderInfoStore.get('wine-releases', [])
-  // use Proton-GE type if on Linux and GamePortingToolkit if on Mac
+  // use GE-Proton type if on Linux and GamePortingToolkit if on Mac
   const release = availableWine.filter((version) => {
     if (isLinux) {
-      return version.version.includes('Proton-GE-Proton')
+      return version.version.includes('GE-Proton')
```

Comment:
wine-ge versions are `Wine-GE-Proton` so this `includes` will match those incorrectly

the idea for this filter is to ignore all `wine-ge` and pick `proton-ge` as a default wine/proton to suggest

maybe this needs to be changed to `startsWith`? or maybe `!version.version.includes('Wine')`?

Reply:
I see
I wonder why we aren't just filtering on `version.type` actually

Reply:
I know at some point  the code was filtering for `wine-ge` but not `-LoL` I think? that's all kinda all now though